.github/workflows/generate-package-list.yml

```diff
@@ -0,0 +1,72 @@
+name: generate-package-list
+on: 
+  workflow_dispatch:
+    inputs:
+      minorRelease:
+        description: 'Minor release (e.g. 1.13.x)'
+        required: true
+        type: string
+      patchRelease:
+        description: 'Patch release (e.g. 1.13.2)'
+        required: true
+        type: string
+      issueNumber:
+        description: 'Github issue number to close (e.g. 139)'
+        type: string
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  generate-package-list:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout docs site
+        uses: actions/checkout@v3
+        with:
+          path: bottlerocket-project-website
+      - name: Checkout bottlerocket os
+        uses: actions/checkout@v3
+        with:
+          repository: bottlerocket-os/bottlerocket
+          path: bottlerocket
+          ref: v${{ inputs.patchRelease }}
+      - name: Remove old package list
+        run: |
+          rm -rf bottlerocket-project-website/content/en/os/$MINOR_RELEASE/version-information/packages/$PATCH_RELEASE/
+          find bottlerocket-project-website/content/en/os/$MINOR_RELEASE/version-information/packages/
+        env:
+          MINOR_RELEASE: ${{ inputs.minorRelease }}
+          PATCH_RELEASE: ${{ inputs.patchRelease }}
```

Comment:
Unsure if this is necessary?

Reply:
`PATCH_RELEASE: ${{ inputs.patchRelease }}`?

Reply:
Removing this step

Reply:
ahh. I get it. Yeah, let's retain the removal